### PR TITLE
Refactor Feature Flags for Reactivity

### PR DESCRIPTION
**Pull request by jguertin**
Combined work with Bryan to begin the process of integrating Launch Darkly.  This step refactors our existing placeholder FF approach to leverage reactivity.  By providing the "global" ff object in the App using `reactive()`, when we inject the object into components and leverage it in templates, the UI will respond immediately to changes in settings.

The following step after this commit will be to replace the internals with Launch Darkly integration.

### CODE REVIEW

**Review by bryan-gilbert — APPROVED** [review]

File: ppr-ui/src/views/Home.vue [hunk 1]
```diff
@@ -35,29 +43,29 @@
...
 export default createComponent({
   setup(): Data {
     const router = useRouter()
-    const featureOne = inject("featureOne", ref(false))
-    const featureTwo = inject("featureTwo", ref(false))
+
+    // Feature Flags
+    const flags = useFeatureFlags()
+    const featureOneLabel = computed((): string => (flags.feature1 ? 'enabled' : ' disabled'))
+    const featureTwoLabel = computed((): string => (flags.feature2 ? 'enabled' : 'disabled'))
```
Comment (WalterMoar) [inline ppr-ui/src/views/Home.vue:62]: Is the difference in the disabled strings intentional?

Comment (WalterMoar) [reply ppr-ui/src/views/Home.vue:62]: Oh... is this a label that we'll throw away? If so then ignore the above.

**Review by WalterMoar — APPROVED** [review]
I haven't had time to look into the Composition API, so not sure if I can give this a good review without doing so. Will try to play with it this week, though.

File: ppr-ui/src/views/Home.vue (hunk 1 above)
Comment (bryan-gilbert) [reply ppr-ui/src/views/Home.vue:62]: Yes this is throw away

**Pull request merged**